tests/nvme_test.py

```diff
@@ -530,6 +538,11 @@ def test_host_nqn(self):
         FAKE_HOSTID1 = 'eeeeeeee-eeee-eeee-eeee-eeeeeeeeeeee'
         FAKE_HOSTID2 = 'beefbeef-beef-beef-beef-beefdeadbeef'
 
+        # libnvme might have been configured with a different prefix than libblockdev
+        sysconf_dir = self._get_sysconf_dir()
+        if sysconf_dir != '/etc':
+            raise unittest.SkipTest("libblockdev was not configured with standard prefix (/usr)")
```

Comment:
Please use `self.skipTest` here (we have `self` here and it produces better output).

Reply:
Fixed.